server/portal/templates/includes/nav_search.html

```diff
@@ -0,0 +1,24 @@
+{# @var className #}
+
+{# Encapsulate search bar as custom element, so styles do not bleed out #}
+{# IDEA: Should CMS use React component or Portal's React use custom element #}
+{# WARNING: Some markup is duplicated in other repositories #}
+{# SEE: https://confluence.tacc.utexas.edu/x/LoCnCQ #}
+<tacc-search-bar class="s-search-bar  {{className}}">
+  <!-- The search bar is populated here (from an external template). -->
+  {# FAQ: Content populated via JavaScript, below #}
+</tacc-search-bar>
+
+<script type="module">
+  import * as importHTML from '/static/site_cms/js/modules/importHTML.js';
+  import * as registerCustomElement from '/static/site_cms/js/modules/registerCustomElement.js';
+
+  importHTML.getFromURL('/cms/nav/search/markup/').then(markup => {
```

Comment:
The markup returned here sets the search contents as `query`, when it should be `query_string`.

To test:
1. run `./manage.py rebuild_index` in `frontera_prtl_cms`
2. run the following in `frontera_prtl_django` shell:
```
from portal.libs.elasticsearch.indexes import setup_files_index
setup_files_index()
```
3. Create a test CMS page and add the string "Test" as content. Be sure to publish.
4. See that results are returned when searching in the Search Page searchbar, and not from the searchbar in the header.

Reply:
I have changed the parameter from `query` to `query_string`.

<details><summary>Proof of Change</summary>

1. `cms-site-template` commit __`36fca5b`__ (on branch `task/FP-531-search-bar-in-header`)
2. `cms-site-template` merge request [!79](https://gitlab.tacc.utexas.edu/wma-cms/cms-site-template/merge_requests/80)
3. `taccwma/frontera-cms` tag __`36fca5b`__ ([manual Jenkins build](https://jenkins01.tacc.utexas.edu/job/Core_CMS/25/console))
4. `Frontera-Portal` commit 5e8be4d (on branch `task/FP-531-search-bar-in-header`)

> P.S. This is also available on CMS repo `master` branch:
>
> 5. `cms-site-template` commit `133be32` (on branch `master`)
> 6. `taccwma/frontera-cms` tag __`133be32`__ ([manual Jenkins build](https://jenkins01.tacc.utexas.edu/job/Core_CMS/26/console))

</details>

But, I failed to successfully search using your instructions, with my environment. _**Disclaimer:** I know very little about elasticsearch._

```
➜  Frontera-Portal git:(task/FP-531-search-bar-in-header) docker exec -it frontera_prtl_django /bin/bash
root@8f414d216b23:/srv/www/portal/server# python
Python 3.6.12 (default, Dec 18 2020, 06:04:50) 
[GCC 8.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from portal.libs.elasticsearch.indexes import setup_files_index
>>> setup_files_index()
Traceback (most recent call last):
  File "/opt/pysetup/.venv/lib/python3.6/site-packages/elasticsearch_dsl/connections.py", line 110, in get_connection
    return self.create_connection(alias, **self._kwargs[alias])
KeyError: 'default'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/srv/www/portal/server/portal/libs/elasticsearch/indexes.py", line 57, in setup_files_index
    index = setup_indexes('files', reindex, force)
  File "/srv/www/portal/server/portal/libs/elasticsearch/indexes.py", line 36, in setup_indexes
    if force or not index.exists():
  File "/opt/pysetup/.venv/lib/python3.6/site-packages/elasticsearch_dsl/index.py", line 414, in exists
    return self._get_connection(using).indices.exists(index=self._name, **kwargs)
  File "/opt/pysetup/.venv/lib/python3.6/site-packages/elasticsearch_dsl/index.py", line 135, in _get_connection
    return get_connection(using or self._using)
  File "/opt/pysetup/.venv/lib/python3.6/site-packages/elasticsearch_dsl/connections.py", line 113, in get_connection
    raise KeyError("There is no connection with alias %r." % alias)
KeyError: "There is no connection with alias 'default'."
```

Reply:
@jarosenb have you seen this error before?